_includes/eventnav.html

```diff
@@ -2,55 +2,53 @@
 {% assign eventdata = page.data %}
 {% assign pagedata = site.data.events[eventdata] %}
   <!-- Navigation-->
-  <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
-    <div class="container">
```

Comment:
I removed the container because the `navbar` should not contains it, otherwise it results in a not-fully-working navbar.